doc/code/Apps.md

```diff
@@ -35,18 +35,20 @@ that will call the method `Refresh()` periodically.
 
 ## App types
 
-There are basically 2 types of applications : **system** apps and **user** apps.
+There are basically 3 types of applications : **system** apps and **user** apps and **watchfaces**.
 
 **System** applications are always built into InfiniTime, and InfiniTime cannot work properly without those apps.
 The watchfaces, settings, notifications and the application launcher are examples of such system applications.
 
 **User** applications are optionally built into the firmware. They extend the functionalities of the system.
 
-The distinction between **system** and **user** applications allows for more flexibility and customization.
-This allows to easily select which user applications must be built into the firmware
+**Watchfaces** are very similar to the **user** apps, they are optional, but at least one must be built into the firmware.
+
+The distinction between **system** apps,  **user** apps and watchfaces allows for more flexibility and customization.
```

Comment:
two word `watch face` and emphasis (as the other two types are emphasized as well
```suggestion
The distinction between **system** apps,  **user** apps and **watch faces** allows for more flexibility and customization.
```